cmd/antrea-agent/agent.go

```diff
@@ -39,9 +39,10 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/version"
 )
 
-// Determine how often we go through reconciliation (between current and desired state)
-// Same as in https://github.com/kubernetes/sample-controller/blob/master/main.go
-const informerDefaultResync time.Duration = 30 * time.Second
+// informerDefaultResync is the default resync period if a handler doesn't specify one.
+// Use the same default value as kube-controller-manager:
+// https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/apis/config/v1alpha1/defaults.go#L120
```

Comment:
It's a nit but this link may no longer be valid if the k8s code is updated. Maybe use https://github.com/kubernetes/kubernetes/blob/release-1.17/pkg/controller/apis/config/v1alpha1/defaults.go#L120 instead. If picks a release but at least you know the link will remain valid.

Reply:
Makes sense, updated.